src/tests/integration/test_foundry_prove.py

```diff
@@ -386,20 +385,20 @@ def test_foundry_resume_proof(
         port=server.port,
         bug_report=bug_report,
     )
-    id = id_for_test(test, prove_res)
 
-    proof = foundry.get_apr_proof(f'{test}:{id}')
+    proof = foundry.get_apr_proof(f'{single(prove_res).id}')
     assert proof.pending
 
     prove_res = foundry_prove(
         foundry_root,
-        tests=[(test, id)],
+        tests=[(test, None)],
         auto_abstract_gas=True,
-        max_iterations=6,
+        max_iterations=10,
```

Comment:
It was intentional. The additional iterations are needed since #54 but I think the test was broken so it wasn't failing properly. This PR should fix that.